desc/profiles.py

```diff
@@ -244,6 +244,17 @@ def __sub__(self, x):
         """Subtract another profile from this one."""
         return self.__add__(-x)
 
+    def __pow__(self, x):
+        """Raise this profile to a power."""
+        if np.isscalar(x):
+            return PoweredProfile(x, self)
+        else:
+            raise NotImplementedError()
```

Comment:
the pythonic thing here would be `return NotImplemented`

Reply:
I followed the format of our existing dunder methods. If we make this change to all of them, then doing stuff like `PowerSeriesProfile() + 4` will raise a `TypeError` instead of the `NotImplementedError`, and we would have to change our tests accordingly. I'm not sure that's what we want.